StarWars.Android/Resources/layout/activity_detail.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+                                       xmlns:local="http://schemas.android.com/apk/res-auto"
+                                       android:layout_width="match_parent"
+                                       android:layout_height="match_parent"
+                                       android:fitsSystemWindows="true"
+                                       android:background="#000000">
+
+    <LinearLayout
```

Comment:
Also I suggest you to try ConstraintLayout. It's better for performance. You can use AndroidStudio to create layout and later copy and paste here